src/System.Management.Automation/namespaces/LocationGlobber.cs

```diff
@@ -3363,13 +3363,11 @@ internal static string RemoveProviderQualifier(string path)
                 StringContainsGlobCharacters(leafElement) ||
                 isLastLeaf)
             {
-                string regexEscapedLeafElement = ConvertMshEscapeToRegexEscape(leafElement);
```

Comment:
Can you explain this change?

Reply:
As explained in the PR Summary, `regexEscapedLeafElement` is passed to `WildcardPattern::Get`, which does not make sense since in PowerShell, regex escaped string is `\[abc\]`, while wildcard escaped string is ``` `[abc`] ```.
Moreover, why should we escape/alter the pattern string in methods `GenerateNew*PathsWithGlobLeaf` which is supposed to glob? This behaviour is causing issue on tab completion when filename contains bracket.

Reply:
Can I mark this resolved?